kpi/serializers/v2/user.py

```diff
@@ -58,26 +66,181 @@ def __get_public_collection_ids(user_id):
+            'public_collections_count',
+            'server_time',
+            'git_rev',
         )
 
+    def get_accepted_tos(self, obj) -> bool:
+        """
+        Verifies user acceptance of terms of service (tos) by checking that the tos
+        endpoint was called and stored the current time in the `private_data` property
+        """
+        try:
+            user_extra_details = obj.extra_details
+        except obj.extra_details.RelatedObjectDoesNotExist:
+            return False
+        accepted_tos = 'last_tos_accept_time' in user_extra_details.private_data.keys()
+        return accepted_tos
+
     def get_asset_count(self, user):
         return user.assets.count()
 
+    def get_current_service_usage(self, user):
+        # Cannot access service usage for anonymous users
+        if user.username == 'AnonymousUser':
+            return None
+
+        serializer = ServiceUsageSerializer(
+            instance=get_database_user(user), context=self.context
+        )
+        return serializer.data
+
+    def get_deployed_asset_count(self, user) -> int:
+        return user.assets.filter(_deployment_status='deployed').count()
+
+    def get_extra_details_uid(self, user):
+        if hasattr(user, 'extra_details'):
+            return user.extra_details.uid
+        return None
+
+    def get_git_rev(self, obj):
+        request = self.context.get('request', False)
+        if constance.config.EXPOSE_GIT_REV or (request and request.user.is_superuser):
+            return settings.GIT_REV
+        else:
+            return False
+
+    def get_gravatar(self, obj):
+        return gravatar_url(obj.email)
+
+    def get_last_login(self, obj):
+        if obj.last_login is not None:
+            return obj.last_login.astimezone(ZoneInfo('UTC')).strftime(
+                '%Y-%m-%dT%H:%M:%SZ'
+            )
+        return None
+
     def get_metadata(self, user):
         if not hasattr(user, 'extra_details'):
             return {}
         return user.extra_details.data
+
+    def get_mfa_is_active(self, user):
+        if hasattr(user, 'mfa_methods'):
+            return user.mfa_methods.filter(is_active=True).exists()
+        return False
+
+    def get_organizations(self, user):
+        if not user.organization:
+            return None
+        else:
+            return {
+                'organization_name': user.organization.name,
+                'organization_uid': (
+                    str(user.organization.id) if user.organization.id else None
+                ),
+                'role': user.organization.get_user_role(user),
+            }
+
+    def get_projects_url(self, obj):
+        return '/'.join((settings.KOBOCAT_URL, obj.username))
+
+    def get_server_time(self, obj):
+        return datetime.datetime.now(tz=ZoneInfo('UTC')).strftime('%Y-%m-%dT%H:%M:%SZ')
+
+    def get_sso_is_active(self, user):
+        return SocialAccount.objects.filter(user=user).exists()
+
+    def get_subscriptions(self, user):
```

Comment:
We need to find another approach if we want to filter by this field. 
See [internal discussion](https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/Extended.20user.20stats.20endpoints.20for.20superusers.20.28cf282a382628.29/near/66632) before doing something with that.